server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/bulkImport.js

```diff
@@ -16,86 +16,97 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-/* JSLint global definitions */
-/*global
-    $, document, sessionStorage, clearTableBody, EMPTY_ROW_THREE_CELLS, EMPTY_CELL, getBulkImports
-*/
```

Comment:
Why did you remove the lint comments?

Reply:
js-lint has conflicting formatting with js-beautify so I figured it was not needed anymore. Do you think it should be added back?